--- gnome-network-1.99.2/desktop/client/gnome-remote-desktop.desktop.in~	Tue Jul 27 20:58:25 2004
+++ gnome-network-1.99.2/desktop/client/gnome-remote-desktop.desktop.in	Tue Jul 27 20:58:25 2004
@@ -8,7 +8,8 @@
 Type=Application
 Version=1.0
 Actions=Full;
-Categories=Network;Application;
+Categories=GTK;GNOME;Network;
+MimeType=application/x-rdp
 Encoding=UTF-8
 
 [Desktop Action Full]
--- gnome-network-1.99.2/gnome-netinfo/gnome-netinfo.desktop.in~	Tue Jul 27 20:54:56 2004
+++ gnome-network-1.99.2/gnome-netinfo/gnome-netinfo.desktop.in	Tue Jul 27 20:57:33 2004
@@ -1,8 +1,12 @@
 [Desktop Entry]
+Encoding=UTF-8
 Name=Network information
+Name[pl]=Informacje o sieci
 Comment=Network information tools
+Comment[pl]=Narzędzia informujące o sieci
 Exec=gnome-netinfo
 Icon=gnome-netinfo.png
-Terminal=0
+Terminal=false
 Type=Application
-Categories=Application;Network;
+Categories=GTK;GNOME;Network;
+# vi: encoding=utf-8
--- gnome-network-1.99.2/network-utilities/gnome-network-utilities.desktop.in~	Tue Jul 27 20:55:38 2004
+++ gnome-network-1.99.2/network-utilities/gnome-network-utilities.desktop.in	Tue Jul 27 20:58:01 2004
@@ -1,9 +1,12 @@
 [Desktop Entry]
+Encoding=UTF-8
 Name=Remote shell
+Name[pl]=Zdalna powłoka
 Comment=Open a shell on a remote server
+Comment[pl]=Uruchomienie powłoki na zdalnym serwerze
 Exec=gnome-remote-shell
 Icon=gnome-remote-shell.png
-Terminal=0
+Terminal=false
 Type=Application
-Categories=Application;Network;
-
+Categories=GTK;GNOME;Network;RemoteAccess;
+# vi: encoding=utf-8
