diff -Nuard gnome-network-1.99.4.origa/configure.in gnome-network-1.99.4/configure.in
--- gnome-network-1.99.4.origa/configure.in	2003-11-25 16:57:17.000000000 +0100
+++ gnome-network-1.99.4/configure.in	2003-11-26 23:10:08.510731408 +0100
@@ -38,8 +38,15 @@
 
 dnl check for rfb (libvncserver) headers.  It could cleaner if libvncserver would
 dnl provide PGK_CONFIG checks.
-AC_CHECK_HEADERS(rfb/rfb.h rfb.h, have_rfb=yes)
-AM_CONDITIONAL(RFB, test "x$have_rfb" != "x")
+AC_PATH_PROG(LIBVNC, libvncserver-config, no)
+
+if test "x$LIBVNC" != "xno"; then
+	AC_CHECK_HEADERS(rfb/rfb.h rfb.h, have_rfb=yes)
+	AM_CONDITIONAL(RFB, test "x$have_rfb" != "x")
+	VNC_LIBS=`libvncserver-config --libs`
+fi
+
+AC_SUBST(VNC_LIBS)
 
 PKG_CHECK_MODULES(ZVT, libzvt-2.0, have_zvt=yes, have_zvt=no)
 AM_CONDITIONAL(HAVE_LIBZVT, test "x$have_zvt" = "xyes")
diff -Nuard gnome-network-1.99.4.origa/desktop/server/Makefile.am gnome-network-1.99.4/desktop/server/Makefile.am
--- gnome-network-1.99.4.origa/desktop/server/Makefile.am	2003-11-25 18:53:29.000000000 +0100
+++ gnome-network-1.99.4/desktop/server/Makefile.am	2003-11-26 23:14:19.740538672 +0100
@@ -1,8 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
 # if libvncserver is not installed, we don't build anything.
-VNC_LIBS=-lXtst -lXext -lX11 -lvncserver -lnsl -lz -lpthread -ljpeg @LIBS@
-
 INCLUDES = \
         -DPACKAGE_DATA_DIR=\""$(datadir)"\" \
         -DPACKAGE_LOCALE_DIR=\""$(prefix)/$(DATADIRNAME)/locale"\" \
@@ -15,5 +13,5 @@
 	tray.c tray.h \
 	x11vnc.c
 
-gnome_desktop_sharing_LDADD = $(GNETWORK_LIBS) @VNC_LIBS@
+gnome_desktop_sharing_LDADD = $(GNETWORK_LIBS) -lXtst -lXext -lX11 @VNC_LIBS@
 
