diff -urN aaa/gnome-network-1.99.3/desktop/data/Makefile.am gnome-network-1.99.3/desktop/data/Makefile.am
--- aaa/gnome-network-1.99.3/desktop/data/Makefile.am	2003-11-25 12:33:26.000000000 +0100
+++ gnome-network-1.99.3/desktop/data/Makefile.am	2003-11-25 15:07:22.000000000 +0100
@@ -5,7 +5,7 @@
 
 schemadir = $(GCONF_SCHEMA_FILE_DIR)
 schema_in_files = gnome-desktop-sharing.schemas.in
-schema_DATA = $(schema_in_files:.schema.in=.schema)
+schema_DATA = $(schema_in_files:.schemas.in=.schemas)
 @INTLTOOL_SCHEMAS_RULE@
 
 install-data-local:
